### DISCUSSION TITLE
overhang/island detection supports + sign pattern

### DISCUSSION THREAD

**Discussion by sstarcher** [sn4k3/UVtools#704] [Q&A] [answered]
Anytime I use a raft and supports it shows a + sign pattern where each support is.  My understanding of this is that the overhang detection is radiating on the X and Y axis outward.  I would think that supports would radiate in all directions and not just 2 cardinal directions.

Am I missing something?   I could increase the distance for overhang, but that still just seems wrong.

![Screen Shot 2023-05-11 at 6 24 46 AM](https://github.com/sn4k3/UVtools/assets/3778974/24d6a737-3e99-479d-84e2-a62a857b6fc0)

**Comment by sn4k3** [comment]
Overhangs detection is based on simple cross erode which is creating those virtual crosses. Need to inspect the file to tell why but looks like is your threshold values that trigger that potential problem.

**Reply by sstarcher** [reply]
I have increased mine fairly significantly from the default.  It just sounds like overhang detection is very limited and does not work well for supported rafts.  The print is perfectly supported, but if I increase the overhang parameter on the X,Y axis it would not actually support such a wide overhang.

Overall it sounds like I should just ignore the overhang detection/manual inspection when using a raft.

**Reply by sn4k3** [reply, marked as answer]
> Overall it sounds like I should just ignore the overhang detection/manual inspection when using a raft.

That's the way. UVtools should be used in coop with its user, you should ignore the false-positives that you know beforehand
